tests/micro/common/conftest.py

```diff
@@ -34,6 +40,11 @@ def pytest_addoption(parser):
     )
 
 
+@pytest.fixture
+def platform(request):
```

Comment:
instead of passing platform, can you detect the platform based on the board name? like this:
https://github.com/apache/tvm/blob/bd800c9cf0963666735680b5927ee5497e4da2a9/tests/micro/common/test_tvmc.py#L50

Reply:
I don't like auto-detecting based on board name, as there are several boards that are compatible with both Zephyr and Arduino. For example, the [`stm32f746g_disco`](https://www.st.com/en/evaluation-tools/32f746gdiscovery.html), the [`nucleo_f746zg`](https://www.st.com/en/evaluation-tools/nucleo-f746zg.html), and the [`nucleo_l4r5zi`](https://www.st.com/en/evaluation-tools/nucleo-l4r5zi.html) are all compatible with [both Zephyr](https://docs.zephyrproject.org/2.6.0/boards/index.html) and [Arduino](https://github.com/stm32duino/Arduino_Core_STM32).